automation/run-tests.sh

```diff
@@ -307,7 +307,7 @@ dump_network_info
 
 pyclean
 if [[ "$CI" != "true" ]];then
-    container_exec "cp -rf $CONTAINER_WORKSPACE /root/nmstate-workspace"
+    container_exec "cp -rf $CONTAINER_WORKSPACE /root/nmstate-workspace || true"
```

Comment:
This does not seem to be correct. If copying failed, the test should not continue.

Reply:
I explained this in the commit. The copy may fail even if the files copy fully. And if there is a problem, the tests will fail later on.

Reply:
can you maybe change this to do this only if `CONTAINER_CMD == docker`?

Reply:
Sure, thanks.

Reply:
Will this suffice?
```bash
    if [ "$CONTAINER_CMD" == "docker" ];then
      container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm' || true
    else
      container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
    fi
```

Reply:
works for me

Reply:
Done

Reply:
This conversation was actually for the cp not the rm

Reply:
Ohh, right. Fixed.